clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -147,7 +147,8 @@ static bool startsNextOperand(const FormatToken &Current) {
 // Returns \c true if \c Current is a binary operation that must break.
 static bool mustBreakBinaryOperation(const FormatToken &Current,
                                      const FormatStyle &Style) {
-  return Style.BreakBinaryOperations != FormatStyle::BBO_Never &&
+  return Current.CanBreakBefore &&
+         Style.BreakBinaryOperations != FormatStyle::BBO_Never &&
```

Comment:
```suggestion
  return Style.BreakBinaryOperations != FormatStyle::BBO_Never &&
          Current.CanBreakBefore &&
```
To have a smaller diff.

Reply:
Done